src/shared/install.c

```diff
@@ -1197,7 +1197,8 @@ static int config_parse_default_instance(
         }
 
         if (!unit_instance_is_valid(printed))
-                return -EINVAL;
+                return log_syntax(unit, LOG_WARNING, filename, line, SYNTHETIC_ERRNO(EINVAL),
+                                  "Invalid DefaultInstance value \"%s\".", printed);
```

Comment:
`DefaultInstance=`, i.e. missing `=`.

Reply:
Yeah, I wasn't sure how to write this. I opted to skip "=" to avoid confusion that the stuff directly following the assignment is the value. The caveat here is that the string in quotes is *not* the actual value written by the user, so saying `Invalid DefaultValue=%s setting` would be even more confusing.

But if you think `"Invalid DefaultInstance= value \"%s\"."` is better, I can change to that.

Reply:
I'd prefer with the `=` suffix. As long as a space follows  I think the ambiguity it creates is limited enough

Reply:
I'll update.